workflow/controller/operator.go

```diff
@@ -2266,9 +2266,9 @@ func (woc *wfOperationCtx) executeContainer(ctx context.Context, nodeName string
 func (woc *wfOperationCtx) getOutboundNodes(nodeID string) []string {
 	node := woc.wf.Status.Nodes[nodeID]
 	switch node.Type {
-	case wfv1.NodeTypeSkipped, wfv1.NodeTypeSuspend:
+	case wfv1.NodeTypeSkipped, wfv1.NodeTypeSuspend, wfv1.NodeTypePod:
```

Comment:
@alexec You changed this in #5099 (https://github.com/argoproj/argo-workflows/pull/5099/files#diff-f321d4af83fcf8311dc80c0d50c59ac4c240f761206e7bb652709870eb9feb43R2260)

I believe that this is the correct code, but I may be missing something about how container sets are linked together. Can you please review?

Reply:
I can't remember.

Reply:
I was wrong, this fix will introduce a small break to container sets. I'll find a fix that works for both.

Reply:
> I'll find a fix that works for both.

Fixed